### PR TITLE
chore: Adding "tests/dependency-convergence" module to release please config

### PR DESCRIPTION
**Pull request by ddixit14**
Adding tests/dependency-converge in config so that release-please updates its pom.xml

### CODE REVIEW

File: tests/dependency-convergence/pom.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.google.cloud</groupId>
-  <artifactId>full-convergence-check</artifactId>
+  <artifactId>full-convergence-check</artifactId><!-- {x-version-update:full-convergence-check:current} -->
```
Comment (suztomo) [inline tests/dependency-convergence/pom.xml:5]: The tag should not be here

**Review by suztomo — APPROVED** [review]
Needs a fix

File: tests/dependency-convergence/pom.xml (hunk 1 above)
Comment (ddixit14) [reply tests/dependency-convergence/pom.xml:5]: This tag is for "full-convergence-check", not "google-cloud-bom". This is just like [tests/pom.xml](https://github.com/googleapis/java-cloud-bom/blob/main/tests/pom.xml#L6). Tag for "google-cloud-bom" already exists in this file.

Comment (ddixit14) [reply tests/dependency-convergence/pom.xml]: Fixed it.

**Pull request merged**